en/news/_posts/2016-12-12-ruby-2-4-0-rc1-released.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: news_post
+title: "Ruby 2.4.0-rc1 Released"
+author: "naruse"
+translator:
+date: 2016-12-12 00:00:00 +0900
```

Comment:
Considering the current local time in Japan I was so bold to change without waiting for feedback.

I used `2016-12-12 18:00:00 +0900`, similar to various other releases, assuming the intention of having 2012-12-12 as date.